frontend/simple/views/Invite.vue

```diff
@@ -103,13 +103,11 @@
 import * as Events from '../../../shared/events'
 import backend from '../js/backend/'
 import { latestContractState } from '../js/state'
-import { HapiNamespace } from '../js/backend/hapi'
+import { namespace } from '../js/backend/hapi'
 import L from '../js/translations'
 
-const namespace = new HapiNamespace()
-
 export default {
-  name: 'InviteView',
+  name: 'Invite',
```

Comment:
**BUG:** shouldn't be able to add members who are already in the group, and definitely shouldn't be able to send them an invite.